tflint/tflint.go

```diff
@@ -0,0 +1,117 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
I think you should instead say here that this code embeds `tflint`, which is under an MPL license, and you can find its source code at `<URL>`.

Reply:
Updated 6e09bc8